src/NorthwindWebsite/Business/Services/Implementations/CategoryService.cs

```diff
@@ -47,6 +51,9 @@ public async Task UpdateCategoryWithPicture(FileUploadDto fileUploadModel)
         await _categoryRepository.Update(category);
     }
 
-    public async Task<byte[]> GetImage(int id) =>
-        await _categoryRepository.GetImage(id);
-}
+    public async Task<byte[]> GetImage(int id)
+    {
+        _logger.LogWarning("Getting image from database.");
+        return await _categoryRepository.GetImage(id);
```

Comment:
Add space before `return`, please.
Also, I advise to separate log calls with empty spaces to avoid it overlapping with other code parts